crates/node_binding/src/resolver_factory.rs

```diff
@@ -26,48 +27,66 @@ impl JsResolverFactory {
     })
   }
 
-  pub fn get_resolver_factory(&mut self, resolve_options: Resolve) -> Arc<ResolverFactory> {
+  pub fn get_resolver_factory(
+    &mut self,
+    resolve_options: Resolve,
+    pnp_manifest_path: Option<PathBuf>,
+  ) -> Arc<ResolverFactory> {
     match &self.resolver_factory {
       Some(resolver_factory) => resolver_factory.clone(),
       None => {
-        let resolver_factory = Arc::new(ResolverFactory::new(resolve_options));
+        let pnp_manifest = pnp_manifest_path
+          .and_then(|x| load_pnp_manifest(x).ok())
+          .map(Arc::new);
```

Comment:
nit: Perhaps this function should return a `Result`, or panic if the PnP file exists but can't be loaded (since presumably it'd cause all resolutions to fail with cryptic "module not found" errors)?

Reply:
thanks for tip, yes, it need some api change